FRAuth/FRAuth/Callbacks/AppIntegrityCallback.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  AppIntegrityCallback.swift
+//  FRAuth
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+
+@available(iOS 14.0, *)
+open class AppIntegrityCallback: MultipleValuesCallback {
```

Comment:
Make prefix with FR to avoid any weird conflicts in the future with other libraries